compose/xui-frontend.yml

```diff
@@ -0,0 +1,63 @@
+---
+version: '2.4'
```

Comment:
don't we have to add xui repo here?
https://github.com/hmcts/ccd-docker/blob/3a8cdbc36de3a0cd332b82721edbef0bd5c23efb/ccd#L12

Reply:
Also, can you verify the you are able to switch to a branch of xui and run the branch please?

Reply:
I have done that but forget to commit....  😆